prov/sockets/src/sock_conn.c

```diff
@@ -427,14 +432,16 @@ struct sock_conn *sock_ep_connect(struct sock_ep_attr *ep_attr, fi_addr_t index)
 
 	conn_fd = ofi_socket(AF_INET, SOCK_STREAM, 0);
 	if (conn_fd == -1) {
-		SOCK_LOG_ERROR("failed to create conn_fd, errno: %d\n", errno);
+		SOCK_LOG_ERROR("failed to create conn_fd, errno: %d\n",
+			       ofi_sockerr());
 		errno = FI_EOTHER;
 		return NULL;
 	}
 
 	ret = fd_set_nonblock(conn_fd);
 	if (ret) {
-		SOCK_LOG_ERROR("failed to set conn_fd nonblocking, errno: %d\n", errno);
+		SOCK_LOG_ERROR("failed to set conn_fd nonblocking, errno: %d\n",
+			       ofi_sockerr());
```

Comment:
I would push sockerr down into the fd_set_nonblock implementation if it's needed for windows

Reply:
Done